lib/nerves_time/rtc/nxp/pca8565.ex

```diff
@@ -93,6 +97,12 @@ defmodule NervesTime.RTC.NXP.PCA8565 do
     minute_bcd = from_integer(date_time.minute)
     hour_bcd = from_integer(date_time.hour)
     day_bcd = from_integer(date_time.day)
+
+    weekday_bcd =
+      Calendar.ISO.day_of_week(date_time.year, date_time.month, date_time.day, :sunday)
+      |> then(fn {day_of_week, 1, 7} -> day_of_week - 1 end)
+      |> from_integer()
+
```

Comment:
[Calendar.ISO.day_of_week](https://hexdocs.pm/elixir/1.14.5/Calendar.ISO.html#day_of_week/4) returns `1..7` for weekdays.

So `day_of_week`, 1..7 should be decriment 1 for devive.

PCA8565, PCF8563 both uses `0..6` for weekdays, 0 is sunday for both device.

refs. 
https://www.nxp.com/docs/en/data-sheet/PCA8565.pdf, sec. 8.4.5
https://www.nxp.com/docs/en/data-sheet/PCF8563.pdf, sec. 8.4.5